server/internal/server/server.go

```diff
@@ -0,0 +1,65 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net"
+
+	"github.com/llm-operator/rbac-manager/pkg/auth"
+	v1 "github.com/llm-operator/user-manager/api/v1"
+	"github.com/llm-operator/user-manager/server/internal/config"
+	"github.com/llm-operator/user-manager/server/internal/store"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
+)
+
+// New creates a server.
+func New(store *store.S) *S {
+	return &S{
+		store: store,
+	}
+}
+
+// S is a server.
+type S struct {
+	v1.UnimplementedUsersServiceServer
+
+	srv *grpc.Server
+
+	store *store.S
+}
+
+// Run starts the gRPC server.
+func (s *S) Run(ctx context.Context, port int, authConfig config.AuthConfig) error {
+	log.Printf("Starting server on port %d\n", port)
+
+	var opts []grpc.ServerOption
+	if authConfig.Enable {
+		ai, err := auth.NewInterceptor(ctx, authConfig.RBACInternalServerAddr, "api.files")
```

Comment:
or create a scope that is always allowed for authenticated users?

```suggestion
		ai, err := auth.NewInterceptor(ctx, authConfig.RBACInternalServerAddr, "api.users.api_keys")
```

Reply:
> or create a scope that is always allowed for authenticated users?

Yeah maybe. The cyclic dependency between `rbac-server` and `user-manage-server` is a bit weird, but I think it's ok.